libraries/dialogs/src/main/java/com/trendyol/uicomponents/dialogs/BaseBottomSheetDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.trendyol.uicomponents.dialogs
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.trendyol.dialog.R
+
+abstract class BaseBottomSheetDialog<DB : ViewDataBinding> : BottomSheetDialogFragment() {
```

Comment:
Dialog and DialogFragment are different types. That's why I think that class name should be renamed with **BaseBottomSheetDialogFragment**

Reply:
Done.